demo/customer-sdk-demo.html

```diff
@@ -18,10 +19,69 @@
     #map {
       height: 100%;
     }
+
+    .container {
+      width: 500px;
+      display: block;
+      border: 1px solid #e2d9c0;
+      margin: auto;
+      margin-top: 10px;
+      padding-top: 15px;
+    }
+
+    .map-container{
+      width: 100%;
+      display: block;
+      height: 360px;
+    }
+
+    .ui-container{
+      display: block;
+      height: 300px;
+    }
+
+    .logs-container{
+      padding: 10px;
+      border: 1px solid gray;
+      background-color: #f9f9f9;
+      margin-bottom: 15px;
+      max-height: 150px;
+      overflow-y: scroll;
+    }
+
+    .log-item{
+      border-bottom: 1px solid #dedede;
+    }
   </style>
 </head>
-<body>
-  <div id="map"></div>
+<body onload="onLoad()">
+  <div class="container">
+    <div class="map-container">
+      <div id="map"></div>
+    </div>
+    <div class="ui-container">
+      <form onsubmit="return false;">
+        <div class="form-group">
+          <label for="order_uuid">Order UUID</label>
+          <input type="text" class="form-control" id="order_uuid">
+        </div>
+        <div class="form-group">
+          <label for="share_uuid">Share UUID</label>
+          <input type="text" id="share_uuid" class="form-control">
+        </div>
+        <div class="form-group">
+          <label for="customer_access_token">Customer Access Token</label>
+          <input type="text" class="form-control" id="customer_access_token">
+        </div>
+        <p class="help-block" style="color: red;">**Any two of the tree values are required.</p>
```

Comment:
tree.
and also actually this is a lie as the server does not support the customer access token yet, so maybe we don't want to tell this.

also to support yosi's branch you might want to add a box for the developer access token (both for good practice and for the fact that we need this for the region.) otherwise. make sure he does so in his commit.
cc @yosiat